### DISCUSSION TITLE
更新日志 - 20230331

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#683] [Announcements]
# 系统：
1. 后台页面增加菜单搜索，方便插件多时找到插件入口（只限默认主题Argon）
2. 优化“在线升级”页面在部分主题下的表现 #674
3. 更新“在线升级”的服务器地址，以提高带宽

# 设备：
* X86：增加AX200无线网卡支持
* R4S/R2S：取消固件Padding，以加快写卡速度（刷TF卡时写入量从2.5GB降低到500MB）
* R4S：回滚了 20230320 版本 USB 相关的补丁

**Comment by oofyuan** [comment]
请问AX200的兼容性如何？比方说AP模式的稳定性。作为客户端连接其他设备的速度等。

**Reply by jjm2473** [reply]
iwlwifi驱动不支持AP模式

**Reply by qwer99630** [reply]
别折腾了开启不了5G模式，2.4Gap只有80M左右速率，还不稳定，用的AX210

**Reply by jjm2473** [reply]
要开AP模式可以用mt7921，mt7915等pcie模块

**Reply by qinhaiming** [reply]
mt7921K什么时候支持啊？

**Reply by jjm2473** [reply]
没相关设备测试，你自己启动以后看看内核日志，有没有关于mt76和firmware的日志，或者开个issue，把内核启动日志贴出来

**Comment by mouse8867** [comment]
R4S使用3月20日版本设置好PPPoE以后不到几小时就会自动在主页显示DNS错误，重置问题依旧，不安装任何其他插件也会出现此问题。什么情况？

**Reply by jjm2473** [reply]
翻墙了？用nslookup检查dns是不是确实有问题。后台页面的“网络”-“网络诊断”就可以执行nslookup

**Comment by 599662746** [comment]
Exsi7虚拟的【iStoreOS 21.02.3 2023032001】固件，提示有新的固件。然后根据提示在线升级到【istoreos-21.02.3-2023033111-x86-64-squashfs-combined.img.gz】，第三步重启的时候，固件没更新上，还是提示固件更新。

虚拟机版本：VMware ESXi, 7.0.3, 20328353
现用固件：iStoreOS 21.02.3 2023032001
提示升级并自动下载的固件：istoreos-21.02.3-2023033111-x86-64-squashfs-combined.img.gz

到官网下载【istoreos-21.02.3-2023033111-x86-64-squashfs-combined.img.gz】手动备份升级依旧升级不了。

**Reply by jjm2473** [reply]
最近没改x86的固件镜像结构，应该不至于手动升级都不行。手动升级有什么错误提示吗？虚拟机的硬盘有没有问题？

**Comment by pfshow** [comment]
> # 系统：
> 1. 后台页面增加菜单搜索，方便插件多时找到插件入口（只限默认主题Argon）
> 2. 优化"在线升级"页面在部分主题下的表现[在线升级-保持设置并保留当前配置 勾不上 #674](https://github.com/istoreos/istoreos/issues/674)
> 3. 更新"在线升级"的服务器地址，以提高带宽
> 
> # 设备：
> * X86：增加AX200无线网卡支持
> * R4S/R2S：取消固件Padding，以加快写卡速度（刷TF卡时写入量从2.5GB降低到500MB）
> * R4S：回滚了 20230320 版本 USB 相关的补丁

是否可以根据温度增加控制风扇自动启停

**Reply by jjm2473** [reply]
看什么机型，pwm或者gpio控制的一般都是自动启停的。x86可能要在bios设置？
